sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/README.md

```diff
@@ -1,13 +1,227 @@
-# Azure Opentelemetry Exporter for Monitor
+# Microsoft Opentelemetry exporter for Azure Monitor
+
+[![Gitter chat](https://img.shields.io/gitter/room/Microsoft/azure-monitor-python)](https://gitter.im/Azure/azure-sdk-for-python)
+
+The exporter for Azure Monitor allows you to export tracing data utilizing the OpenTelemetry SDK and send telemetry data to Azure Monitor for applications written in Python.
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor) | [Package (PyPi)][pypi] | [API reference documentation][api_docs] | [Product documentation][product_docs] | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/samples) | [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/CHANGELOG.md)
+
+> **NOTE**: This is the preview for the next major version of the [`opentelemetry-azure-monitor-python`](https://github.com/microsoft/opentelemetry-azure-monitor-python).
 
 ## Getting started
 
+### Install the package
+
+Install the Microsoft Opentelemetry Exporter Azure Monitor client library for Python with [pip][pip]:
+
+```Bash
+pip install microsoft-opentelemetry-exporter-azuremonitor --pre
+```
+
+### Prerequisites: 
+To use this package, you must have:
+* Azure subscription - [Create a free account][azure_sub]
+* Azure Monitor - [How to use application insights][application_insights_namespace]
+* Opentelemetry SDK - [Opentelemtry SDK for Python][ot_sdk_python]
+* Python 3.5 or later - [Install Python][python]
+
+### Authenticate the client
+
+Interaction with Azure monitor exporter starts with an instance of the `AzureMonitorSpanExporter` class. You will need an **instrumentation_key** or a **connection_string** to instantiate the object.
+The priority of which value takes on the instrumentation key is:
+* Key from explicitly passed in connection string
+* Key from explicitly passed in instrumentation key
+* Key from connection string in environment variable
+* Key from instrumentation key in environment variable
+Please find the samples linked below for demonstration as to how to authenticate via either approach.
+
+#### [Create Exporter from connection string][sample_authenticate_client_connstr]
+
+```Python
+from microsoft.opentelemetry.exporter.azuremonitor import AzureMonitorSpanExporter
+exporter = AzureMonitorSpanExporter(
+    connection_string = os.environ["AZURE_MONITOR_CONNECTION_STRING"]
```

Comment:
I believe the original sdks used `APPLICATIONINSIGHTS_CONNECTION_STRING` and `APPINSIGHTS_INSTRUMENTATIONKEY`?

Reply:
yup, we changed these in the samples, so just being consistent

Reply:
Wouldn't this be a breaking change if user's are instrumented with both old SDKs and OT sdk? That means they would have to have 2 environment variables to represent the same thing.

Reply:
fair enough - changed it